src/rpc-call-examples/Starknet/call.hurl

```diff
@@ -0,0 +1,41 @@
+PARAMETERS:
+
+request — the details of the function call:
+    contract_address — the address of the contract to be called.
+    entry_point_selector — the selector of the function to be called.
+        calldata — the parameters passed to the function.
+block_id — the hash of the requested block, or number (height) of the requested block, or a block tag, for the block referencing the state or call the transaction on.
+
```

Comment:
Specify which call is this? for ex

balance_of method to address xxx with params yyy,zzz